dateparser/custom_language_detection/fasttext.py

```diff
@@ -0,0 +1,32 @@
+import fasttext
+import os
+
+from dateparser_cli.fasttext_manager import fasttext_downloader
+from dateparser_cli.utils import date_parser_model_home, check_data_model_home_existance
+
+check_data_model_home_existance()
+
+_supported_models = ["large.bin", "small.bin"]
+_downloaded_models = os.listdir(date_parser_model_home)
+
+_model_path = None
+
+for downloaded_model in _downloaded_models:
+    if downloaded_model in _supported_models:
+        _model_path = date_parser_model_home + "/" + downloaded_model
+
+if not _model_path:
+    fasttext_downloader("small")
+
+_language_parser = fasttext.load_model(_model_path)
```

Comment:
@gavishpoddar maybe there was already a comment about this - I think we shouldn't download the model on import, it would be better to download it only if it's used.

Reply:
Thanks, I have done that.